### ISSUE TITLE
Design for Quality Filter Responses on Validation Service UI

### ISSUE THREAD

**Issue by RakshindaAslam** [CDCgov/prime-reportstream#6312]
## Problem statement

As a user testing HL7 data to validate if it will be acceptable, I would like to see the quality filter responses on the UI in an easy to understand format.

## What you need to know

 [Links to documents, workflows, or list of key people and contact info]

- [Figma doc for validation work](https://www.figma.com/file/b2Zd1xzmp4cdwFO8O7edcI/CSV-Uploader-%26-Validation?node-id=2288%3A1127)

## Acceptance criteria

- [ ] Define selection criteria for applying quality filters
- [x] Define how the filter responses will be laid out/categorized on the UI
- [x] Wireframes, if needed

## To do

- [x] Get a list of all quality filter responses

- [x] Meet with James Gilmore to discuss QF responses and what they mean in plain English.

**Comment by RakshindaAslam** [comment]
cc @a-chappelear @bgantick

**Comment by RakshindaAslam** [comment]
@HeatherRienks @bgantick  - how can we get a list of quality filter responses?

**Comment by HeatherRienks** [comment]
@RakshindaAslam - I can check with the O&O team as a starting point.
cc: @bgantick

**Comment by HeatherRienks** [comment]
@GilmoreA6 - we will be reaching out to set up some time to talk about these responses! THanks!

**Comment by a-chappelear** [comment]
Flow may be changed as needed per feedback, but the suggested workflow in the UI is: 

1) Upload file
2) Checks for warnings are made against the RS schema. If a user fails the Hl7 structural check, a user will need to correct those before it goes on to checking quality filters.
3) Once errors are fixed and revalidated, there is a confirmation you passed the schema. RS proceeds to check quality filters and you will either pass or fail those.
4) If a user corrects all errors, they see a confirmation all have passed.

[Updates are annotated in the Figma file](https://www.figma.com/file/b2Zd1xzmp4cdwFO8O7edcI/CSV-Uploader-%26-Validation?node-id=2617%3A2068) _(they are numbered and listing them here as well because there are 2 different design tickets in these wireframes 6312 & 6365.)_

14- Updated copy for the Error alert.
17 - Updated copy for passing the RS schema
16 - New error alert with copy for failing a jurisdiction
19 - Each org/jurisdiction is grouped in it’s own table w/  its header 
20 - Success alerts + copy for passing all checks


cc: @RakshindaAslam @bgantick @HeatherRienks
